### ISSUE TITLE
Support including of "local" properties files in vs201x backend

### ISSUE THREAD

**Issue by vadz** [vslavik/bakefile#82]
I find it very useful to (currently manually) add something like this to the generated projects:

```
  <ImportGroup Label="PropertySheets">
    <Import Project="local.props" Condition="exists('local.props')" />
  </ImportGroup>
```

As this allows me to define local (i.e. machine/user-specific) options such as include/library paths in a separate file.

I'd like bakefile to do this by default because I don't really see any drawbacks in doing this and it's quite useful in practice. Questions:
1. Does anybody else see any drawbacks?
2. What should be the name of this file? `local.props` is obviously not unique, would `$target_local.props` be better?
3. Should there be also perhaps two such imports, one for common `$bakefile_local.props` and second project-specific one? Or would this be an overkill?
4. Should there be any way to customize this? Personally I often want to have these files outside of the VCS repository and so it would be nice to include `../../../local.props` or something like this, but I can live without it. Assuming others think it's worth doing, I guess the way to do it would be via a format-specific `local_props_file` option?

I'm obviously ready to do this myself, but I'd like to hear more opinions about this first. TIA!

**Comment by vslavik** [comment]
1: Not me.
The rest: I think this should be *only* customizable and require explicitly setting a property (or doing something) in the bkl file. That eliminates 2+3.

For example, I do this (manually, currently, because I never got around to implement):

    <ImportGroup Label="ExtensionTargets">
      <Import Project="WinSparkle.targets" />
    </ImportGroup>

That’s slightly different — targets, not properties — but I have use for props too (for build customization settings) and it seems to be both should be handled the same.

**Comment by vadz** [comment]
If we want to make this fully customizable, I'm not sure how to fit it into the existing syntax. Something like

    vs2015.propertysheets = local;
    vs2015.extensiontargets = WinSparkle;

could be enough for the simplest case, but how to specify the `exists` (or another?) condition or even just the label?

**Comment by vslavik** [comment]
> but how to specify the exists (or another?) condition

`property-sheets` and `optional-property-sheets`?

> or even just the label?

? Isn’t the label hard-coded (“PropertySheets” and “ExtensionTargets” respectively)?

**Comment by vadz** [comment]
You can use a custom `Label="My Local Properties"` which may be useful when viewing this in the property manager. As I said, not really critical, but once we start allowing customizing this, it's difficult to know where to stop.

**Comment by vadz** [comment]
This was implemented back in 146b5779d0060577c313295eb9ec8820b282e9e9
